app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>User ID is <%= @user %></h1>
\ No newline at end of file
+
+	<%= render 'user', user:@user	%>
+
+	<div class='bio-main-div'>
+
+
+	<h2>Bio</h2>
+		<%= @user.bio%>
+	</div>
+
+	<%= render @user.posts	%>
+
+<%= link_to user_posts_path(@user) do %>
+<button>See All Posts</button>
+<%end%>
```

Comment:
In the User Show page, it should display the Post number with a particular post. Have a look at the screen shot. ⬇️

| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://github.com/MosDevx/BlogApp/assets/33268927/fd79d00a-0b6f-48d4-b118-4a7d38c9d14f) | ![image](https://github.com/MosDevx/BlogApp/assets/33268927/4d40eb10-9886-471c-be2a-a5c69fd3a495) |

Reply:
Implemented